tests/api/v1/datadog/cassettes/TestUsageGetBillableSummaryErrors/400_Bad_Request.freeze

```diff
@@ -0,0 +1 @@
+2020-07-01T20:02:54.869688-03:00
```

Comment:
Since the 400 Bad Request part of the "TestUsageGetBillableSummaryErrors` test is commented out, we shouldn't need this cassette anymore